src/ut/diameter_handlers_test.cpp

```diff
@@ -736,7 +712,8 @@ TEST_F(DiameterHandlersTest, PPRMainline)
     .WillOnce(DoAll(InvokeArgument<1>(), InvokeArgument<0>()));
 
   // And notify Sprout
-  ppr_sprout_connection(IMPU, IMS_SUBSCRIPTION, HTTP_OK);
```

Comment:
Like that we're now actually going to be doing UT here, not using a real sprout connection to generate some body we then expect to be sent.